Taken from upstream SVN repository to make lintian happy.

Index: lltag.1
===================================================================
--- a/lltag.1	(révision 608)
+++ b/lltag.1	(révision 609)
@@ -863,7 +863,7 @@
 to match %.
 
 .RE
-Additionaly, while renaming, the following codes are available:
+Additionally, while renaming, the following codes are available:
 .RS
 
 .I "%F"
@@ -953,31 +953,31 @@
 .RE
 Show all tags for each OGG files in the current directory:
 .RS
-lltag -S *.ogg
+lltag \-S *.ogg
 .RE
 Show only a selected list of tags for all files in all subdirectories:
 .RS
-lltag --show-tags artist,album,title,number -R .
+lltag \-\-show-tags artist,album,title,number \-R .
 .RE
 Set an arbitrary tag in a file (only works with OGG vorbis or FLAC files):
 .RS
-lltag --tag foo=nil foo.ogg
+lltag \-\-tag foo=nil foo.ogg
 .RE
 Delete the foo tag from a file:
 .RS
-lltag --tag foo= bar.ogg
+lltag \-\-tag foo= bar.ogg
 .RE
 Set the ALBUM, ARTIST and GENRE tag values of the MP3 files in the current directory:
 .RS
-lltag --ARTIST "Queen" --ALBUM "Innunendo" --GENRE "rock" --COMMENT="very cool" *.mp3
+lltag \-\-ARTIST "Queen" \-\-ALBUM "Innunendo" \-\-GENRE "rock" \-\-COMMENT="very cool" *.mp3
 .RE
 Rename a file by assembling its current NUMBER, ARTIST and TITLE tag values:
 .RS
-lltag --no-tagging --rename "%n - %a - %t" foobar.ogg
+lltag \-\-no\-tagging \-\-rename "%n - %a - %t" foobar.ogg
 .RE
 Clear all tags in all FLAC files:
 .RS
-lltag --clear *.flac
+lltag \-\-clear *.flac
 
 
 
Index: Changes
===================================================================
--- a/Changes	(révision 608)
+++ b/Changes	(révision 609)
@@ -1,3 +1,7 @@
+
+  * Fix miscellaneous typos everywhere, reported by Debian's lintian.
+
+
 lltag (0.14.3)
 
   * Add -T and --preserve-time, and the preserve_time configuration
@@ -342,7 +346,7 @@
 
 lltag (0.7.2)
 
-  * Fix compilation and installation when DESTDIR is overriden.
+  * Fix compilation and installation when DESTDIR is overridden.
   * Fix make uninstall.
   * Add a VERSION file.
   * Update all contact addresses and URLs since the project is
@@ -408,7 +412,7 @@
   * When a field appears multiple times in the format, lltag does not print all matched values.
     It checks that they are identical, prints a warning if not, and keeps the first one.
   * Do not tag when there's nothing to tag (fix bogus mp3info/vorbiscomment invocation).
-  * Add comments at the begining of formats.
+  * Add comments at the beginning of formats.
   * Fix matching in man page.
 
  -- Brice Goglin <Brice.Goglin@ens-lyon.org>  Sat, 30 Jul 2005 13:26:54 +0200
Index: README
===================================================================
--- a/README	(révision 608)
+++ b/README	(révision 609)
@@ -21,7 +21,7 @@
 on what type of files you want to work on, you'll need to install
 either the Perl MP3::Tag module (for MP3 files) or mp3info (for MP3 files,
 without ID3v2 support), vorbiscomment (for OGG) or metaflac (for FLAC).
-Additionaly, when using CDDB features, LWP (the libwww-perl module) is
+Additionally, when using CDDB features, LWP (the libwww-perl module) is
 required.
 These dependencies are automatically installed when installing
 Debian or Gentoo prepackaged versions, or at least recommended.
